docusaurus.config.js

```diff
@@ -5,7 +5,9 @@ import autocolorPlugin from "./src/plugins/rehype/autocolor.js";
 const GITHUB_ORG = "comcode-org";
 const GITHUB_PROJECT = "hackmud_wiki";
 
-const USING_ALGOLIA = typeof process.env.ALGOLIA_APPID === "string";
+const USING_ALGOLIA =
+  typeof process.env.ALGOLIA_APPID === "string" &&
+  process.env.ALGOLIA_APPID.length > 0;
```

Comment:
Any concern with just using a truthy here?

```suggestion
const USING_ALGOLIA = !!process.env.ALGOLIA_APPID;
```

Reply:
Either or is fine. Just initially wanted to make the behaviour explicit.

Reply:
I looked it up. The type of all variables under process.env are always "string" -- that check will always be true. Let's use my truthyness check.